.gitlab/newell.gitlab-ci.yml

```diff
@@ -0,0 +1,54 @@
+
+cleanup:
+ stage: .pre
+ extends:
+  - .SVC-Account-Cleanup
+
+hykkt-test-newell:
+  stage: test
+  needs: []
+  variables:
+    SLURM_Q: "newell_shared"
+    MY_CLUSTER: "newell"
+  extends:
+  - .pnnl_tags_template
+  - .hykkt_script_template
+
+
+.report-status:
+  image: mrnonz/alpine-git-curl:alpine3.16
+  variables:
+    GIT_STRATEGY: none
+    STATUS_PROJECT: ORNL/hykkt
+    STATUS_NAME: CentOS-8-ppc64le-gcc/8.5.0-cuda-11.4-v100-arch/70
```

Comment:
```suggestion
    STATUS_NAME: (Newell) CentOS8/ppc64le/gcc@8.5.0/cuda@11.4/V100/cuda_arch=70
```

Reply:
The reason I dropped he newell is that the name gets to long for the github check and you cant see all characters and descriptions. There seems to be a character limit on the status name for github.